lore/features/base.py

```diff
@@ -48,13 +57,22 @@ def version(self, version=str(datetime.date.today())):
         :param version:
         :return:
         """
-        return version
+        return 'v1'
```

Comment:
I think we if the version is not passed by default it would be better to have it scoped by date so that we don't overwrite features. Is there any specific reason you set it to v1?